pygmt/src/sphdistance.py

```diff
@@ -22,7 +22,7 @@
     V="verbose",
 )
 @kwargs_to_strings(I="sequence", R="sequence")
-def sphdistance(table, **kwargs):
+def sphdistance(data=None, x=None, y=None, **kwargs):
     r"""
     Create Voroni polygons from lat/lon coordinates.
```

Comment:
Typo here. Also, just noticed that this summary doesn't seem correct. It seems to suggest Voronoi polygons are created, but the output is a raster grid, not a vector polygon.
```suggestion
    Create Voronoi polygons from lat/lon coordinates.
```